app/api/schema/attendees.py

```diff
@@ -47,7 +47,9 @@ class Meta:
     ticket_id = fields.Str(allow_none=True)
     is_checked_in = fields.Boolean()
     checkin_times = fields.Str(allow_none=True)
+    checkout_times = fields.Str(allow_none=True)
     attendee_notes = fields.Str(allow_none=True)
+    is_checked_out = fields.Boolean()
```

Comment:
is_checked_out = not is_checked_in. We already have `is_checked_in` in the schema. Why add a redundant one ?

Reply:
I thought about it, but I guess when `is_checked_in` is true and `is_checked_out` is false, the person is currently attending the event?

Reply:
I am not sure about it. I think `is_checked_in` = true implies currently attending and if it is false, then it means not attending currently. 

@iamareebjamal Can you please confirm ?

Reply:
Not attending can have 2 states - not arrived and, arrived but currently absent?

Reply:
Yeah, both being false can also be interpreted as if the attendee is/was absent. Although, I have few concerns, 
first why the name is plural "checkin_times"? I believe it will have only one timestamp right?
second, why you even need a flag for it? Simply, checked in timestamp being null represents that attendee never checked in and not null as the timestamp of check-in. This although affects the frontend as well if they are already dependent on flags but will make it easier and redundant free.